Tasks/DeployAzureResourceGroup/AzureResourceManagerHelper.ps1

```diff
@@ -43,7 +43,7 @@ function Create-AzureResourceGroup
 
                 foreach($error in $deploymentError)
                 {
-                    Write-Verbose -Verbose $error
+                    Write-Host (Get-LocalizedString -Key "'{0}'" -ArgumentList $error)
```

Comment:
This error will come from platform, then I don't think we can localize that. If required we can directly print in Host as well